File: app/src/main/java/io/github/anvell/stackoverview/model/Answer.java
package io.github.anvell.stackoverview.model;

import com.google.gson.annotations.SerializedName;

public class Answer {
    @SerializedName("owner")
    public Owner owner;

    @SerializedName("is_accepted")
    public Boolean isAccepted;

    @SerializedName("creation_date")
    public int creationDate;

    @SerializedName("answer_id")
    public int answerId;

    @SerializedName("link")
    public String link;

    @SerializedName("body")
    public String body;
}

File: app/src/main/java/io/github/anvell/stackoverview/model/Answer.kt
package io.github.anvell.stackoverview.model

import com.google.gson.annotations.SerializedName

data class Answer (
        @SerializedName("owner") val owner: Owner,
        @SerializedName("is_accepted") val isAccepted: Boolean,
        @SerializedName("creation_date") val creationDate: Int,
        @SerializedName("answer_id") val answerId: Int,
        @SerializedName("link") val link: String,
        @SerializedName("body") val body: String
)